src/layouts/Media/Media.tsx

```diff
@@ -264,7 +286,10 @@ export const Media = (): JSX.Element => {
                         Deselect all
                       </Button>
                     </Box>
-                    <Box mt="auto">
+                    <Box
+                      mt="auto"
+                      id="isomer-media-onselect-feature-tour-step-1"
```

Comment:
for the ids here (and below), could we use `MEDIA_ONSELECT_FEATURE_STEPS.target`?

Reply:
Fixed in 85f50571.